src/index.js

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root'),
-);
```

Comment:
- [ ] You shouldn't have deleted the App component and created a class with the name of App inside the index.js file. The `index.js` should be present as it was and the App.js file should be present as it was only as a class-based component and not a function-based component.

Reply:
It is done

Reply:
Nice job! This issue is well addressed. :clap: :+1: :+1: